source/account_structure/index.html.md.erb

```diff
@@ -35,6 +28,12 @@ both GOV.UK Pay and the PSP.
 In this setup, payments to services can go to separate bank accounts because
 each service has a PSP account.
 
+You can:
+
+- give access for each service to different staff members
+- report separately on each of your services
+- optionally use different names on the GOV.UK Pay payment pages
```

Comment:
Should we clarify - 'different service names'?

Reply:
Good plan - added!